lib/util/fs/index.spec.ts

```diff
@@ -461,4 +463,43 @@ describe('util/fs/index', () => {
       expect(await readSystemFile(path)).toEqual(Buffer.from('foobar'));
     });
   });
+
+  describe('getFileContentMap', () => {
+    it('reads list of files from local fs', async () => {
+      const fileContentMap = {
+        file1: 'foobar',
+        file2: 'foobar2',
+      };
+
+      await fs.outputFile(`${localDir}/file1`, fileContentMap.file1);
+      await fs.outputFile(`${localDir}/file2`, fileContentMap.file2);
+      const res = await getFileContentMap(Object.keys(fileContentMap), true);
+      expect(res).toStrictEqual(fileContentMap);
+    });
+
+    it('reads list of files from git', async () => {
+      const fileContentMap = {
+        file1: 'foobar',
+      };
+
+      const repo = Git(tmpDir);
```

Comment:
can we mock git for faster tests?

Reply:
Good catch, I should have done that in the first place... :see_no_evil: